CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/home/HomeViewModel.kt

```diff
@@ -49,12 +49,14 @@ class HomeViewModel @Inject constructor(
 
     private fun fetchUsers() {
         showLoading()
-        viewModelScope.launch {
-            when (val result = getUsersUseCase.execute()) {
-                is UseCaseResult.Success -> _users.value = result.data
-                is UseCaseResult.Error -> _error.emit(result.exception.message.orEmpty())
+        execute(kotlin.run {
```

Comment:
🤔  `kotlin.run`? 🤔  it seems a bit odd for me unless we are having a namespace conflict

Reply:
@sleepylee Yes, no need for the prefix `kotlin.` . I'll remove it.